packages/react-union/src/scanning/index.js

```diff
@@ -0,0 +1,18 @@
+import { mergeData } from '../utils';
+
+import * as DOM from './dom';
+import * as Cheerio from './cheerio';
+
+const { getWidgetDescriptors, getCommonDescriptors } = process.env.BROWSER ? DOM : Cheerio;
```

Comment:
I'll change this condition to something like `typeof document === "undefined"` so scanning would work without react-union-scripts